shared/chat/conversation/list-area/normal/index.desktop.js

```diff
@@ -44,34 +45,76 @@ class Thread extends React.PureComponent<Props, State> {
   // When we're triggering scrolling we don't want our event subscribers to fire so we increment this value
   _ignoreScrollToBottomRefCount = 0
   // last height we saw from resize
-  _scrollHeight = 0
+  _scrollHeight: number = 0
 
-  _scrollToBottom = () => {
+  _logIgnoreScroll = (name, fn) => {
+    const oldIgnore = this._ignoreScrollToBottomRefCount
+    fn()
+    if (this.props.debug) {
+      logger.debug('SCROLL', name, 'ignoreScroll', oldIgnore, '->', this._ignoreScrollToBottomRefCount)
+    }
+  }
```

Comment:
Debug is only enabled in story mode right? Could probably move `debug` into a local const in this file `const debug = __STORYBOOK__` (i.e. easily flippable if we need to debug in dev env) and use it like this:

```
  _logIgnoreScroll = debug
    ? (
      (name, fn) => {
        const oldIgnore = this._ignoreScrollToBottomRefCount
        fn()
        if (this.props.debug) {
          logger.debug('SCROLL', name, 'ignoreScroll', oldIgnore, '->', this._ignoreScrollToBottomRefCount)
        }
      }
    ) : (
        (name, fn) => fn()
    )
```

It's not that much difference though, so feel free to just leave it like this.

Reply:
I did it this way because sometimes I actually do want to turn it on in a local (non-storybook) build.

Reply:
oh, i see what you're saying, hmm...

Reply:
done!

Reply:
Yeah; with local dev env you could change it to `const debug = true || __STORYBOOK__`. But these logging functions aren't expensive at all, so probably not worth going through these.